fineract-provider/dependencies.gradle

```diff
@@ -92,7 +92,10 @@ dependencies {
                 // Once we've switched to Java 8 this dep can be removed.
                 //[group: 'com.google.code.findbugs', name: 'jsr305', version: '3.0.0']
                 [group: 'org.springframework', name:'spring-jms'],
-                [group: 'org.apache.activemq', name: 'activemq-broker']
+                [group: 'org.apache.activemq', name: 'activemq-broker'],
+				
+				//jso simple 
+				[group: 'com.googlecode.json-simple', name: 'json-simple', version: '1.1']
```

Comment:
Please refactor usage of json-simple to GSON library. This will lead to ambiguity during imports and non-standardization of json parsing in the platform.